src/test/java/org/minions/devfund/danielmontecinos/moviedatabase/MovieDatabaseTest.java

```diff
@@ -0,0 +1,168 @@
+package org.minions.devfund.danielmontecinos.moviedatabase;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Tests MovieDatabase class.
+ */
+public class MovieDatabaseTest {
+    private MovieDatabase movieDatabase;
+
+    /**
+     * Creates a MovieDatabase object.
+     */
+    @Before
+    public void setUp() {
+        movieDatabase = new MovieDatabase();
+    }
+
+    /**
+     * Tests movieList is empty when no movie is added.
+     */
+    @Test
+    public void testGetMovieListIsEmptyWhenNoMoviesAreAdded() {
+        assertTrue(movieDatabase.getMovieList().isEmpty());
+    }
+
+    /**
+     * Tests actorList is empty when no movie is added.
+     */
+    @Test
+    public void testGetActorListIsEmptyWhenNoMoviesAreAdded() {
+        assertTrue(movieDatabase.getActorList().isEmpty());
+    }
+
+    /**
+     * Tests addMovie method add movie names and movie actors.
+     */
+    @Test
+    public void testAddMovieMethodAddsANewMovieWithAListOfActors() {
+        movieDatabase.addMovie("Movie 1", new String[]{"Actor 1", "Actor 2", "Actor 3"});
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Actor 1" 9 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Actor 3" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Actor 2" 8 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Movie 1" 7 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)